src/GlobalStyle.js

```diff
@@ -31,19 +31,10 @@ const GlobalStyle = createGlobalStyle`
     --box-shadow-keyboard: inset -3px -2px 5px  #e2e2e5;
   }
   body {
-    background-color: black;
-    margin: 0;
-    height: 100vh;
-  
+    background-color: black;   
```

Comment:
```suggestion
    background-color: var(--black);   
```
Note: I remember Jerry saying that pitch black and pure white is not really good, better use a minimal brighter black and minimal darker white; this can easily be done via through your variables :)